use-cases.html

```diff
@@ -48,6 +48,10 @@
             </p>
             <br/>
             <br/>
+            <div style="text-align: center">
```

Comment:
This should be moved down, as right now it looks like part of Dev copies and not bug fixes.

Reply:
Agreed - other than that, looks good to me.